data/bootcamp.yml

```diff
@@ -1,33 +1,26 @@
 - name: Intro + Installation
   teachers: Kevin Bi
-  document: TBD
+  document: https://docs.google.com/presentation/d/1msGwqdzmufFwE7_3fwf807AbDv6ZDZ5Ao0Apqhl38U0
```

Comment:
Actually, this is the presentation for the intro meeting, not the bootcamp. Can we change it to reflect that?